src/objects/zcl_abapgit_objects.clas.abap

```diff
@@ -656,9 +657,10 @@ CLASS zcl_abapgit_objects IMPLEMENTATION.
             lv_path = <ls_result>-path.
           ENDIF.
 
+          ls_item-devclass = lv_package.
```

Comment:
hmm, moving this statement seems like a hack, I guess its not the way the xml gets the package

Reply:
the ls_item is passed to the constructor (ms_item) which includes devclass. it's confusing if this does not match the package passed to deserialize method. 🤷

Reply:
https://github.com/abapGit/abapGit/blob/main/src/objects/aff/zcl_abapgit_object_common_aff.clas.abap#L153